test/dotcom_web/controllers/schedule/timetable_controller_test.exs

```diff
@@ -214,6 +214,14 @@ defmodule DotcomWeb.ScheduleController.TimetableControllerTest do
 
       assert ["1", "2", "3", "4", ^special_stop_id, "5", "6"] = Enum.map(trip_stops, & &1.id)
     end
+
+    test "doesn't crash when missing canonical route patterns", %{conn: conn} do
+      expect(RoutePatterns.Repo.Mock, :by_route_id, fn _, _ ->
+        []
+      end)
+
+      assert build_timetable(conn, @schedules)
```

Comment:
Non-blocking: Is there a way to make a non-trivial relevant empty-state assertion for this use case? Like that it pulls a route pattern from somewhere else, or that it has a reasonable empty state (empty list, `"No route pattern exists you bozo"` string, or something like that?)

Reply:
Yeah... a followup to test the "no trips here" version of the page will do